ui/components/app/snaps/keyring-snap-removal-warning/keyring-snap-removal-warning.tsx

```diff
@@ -156,7 +157,7 @@ export default function KeyringRemovalSnapWarning({
                   inputProps={{
                     'data-testid': 'remove-snap-confirmation-input',
                   }}
-                  type="text"
+                  type={TextFieldType.Text}
```

Comment:
Because we are changing the architecture of this component quite substantially let's use the deprecated JS version for all instances. Then replace them one by one so we can check that all of the inputs still work.

Reply:
hey @georgewrmarshall, if im not wrong we just have to update the `type` prop in files, right? can i work on these changes?

Reply:
Hey @dhruvv173, thanks so much for getting us this far! I think @garrettbear is going to take it from here as it's a blocker for us but it would be great if you could take on these 2 issues:
- https://github.com/MetaMask/metamask-extension/issues/20155
- https://github.com/MetaMask/metamask-extension/issues/20159
We are super close to migrating 100% of the component-library to TypeScript and the above issues will be contributing to that